middlewares/auth.middleware.js

```diff
@@ -0,0 +1,65 @@
+import jwt from 'jsonwebtoken';
+import {
+  UnAuthorizedError,
+  NotFoundError,
+  ForbiddenError,
+} from '../helpers/errors.js';
+import { roles } from '../constants/index.js';
```

Comment:
do we have it already?

Reply:
My mistake, agree) I'll fix it. Thank you